specification/cdn/resource-manager/Microsoft.Cdn/stable/2023-05-01/examples/Profiles_CanMigrate.json

```diff
@@ -13,14 +13,18 @@
   "responses": {
     "200": {
       "body": {
-        "canMigrate": true,
-        "defaultSku": "Standard_AzureFrontDoor",
-        "errors": null
+        "id": "/subscriptions/27cafca8-b9a4-4264-b399-45d0c9cca1ab/resourcegroups/yaoshitest/providers/Microsoft.Cdn/operationresults/e0da7b89-5a5d-4e10-b1d7-2fd0dbd52315/profileresults/DummyProfile/canmigrateresults/DummyProfile",
```

Comment:
I don't see the "id" or "type" properties defined in the response of "Profiles_CanMigrate". Should they be there or is this example in error?

Reply:
Thanks for your review. I updated the schema of "CanMigrate".